pallets/liquidity-mining/src/tests/update_global_farm.rs

```diff
@@ -304,3 +304,36 @@ fn update_global_farm_should_fail_when_yield_period_is_zero() {
 		});
 	});
 }
+
+#[test]
+fn claim_rewards_should_work_after_update_global_farm() {
+	predefined_test_ext_with_deposits().execute_with(|| {
+		let _ = with_transaction(|| {
+			//Arrange
+			let planned_yielding_periods: BlockNumber = 1_000u64;
+			let yield_per_period = Perquintill::from_percent(20);
+			let min_deposit = 20_000;
+
+			set_block_number(100_000);
+
+			//Act
+			assert_ok!(LiquidityMining::update_global_farm(
+				GC_FARM,
+				planned_yielding_periods,
+				yield_per_period,
+				min_deposit,
+			));
+
+			set_block_number(1_000_000);
+
+			//Assert
+			std::assert_eq!(
```

Comment:
Why `std::assert_eq`?
```suggestion
			assert_eq!(
```